busines_logic/student.py

```diff
@@ -0,0 +1,14 @@
+class Student:
+    def __init__(self, username: str, scores: list[float]):
+        self.username = username
+        self.scores = scores
```

Comment:
It is a common practice do denote internal class attributes with leading `_` (e.g. `_username`). Internal attribute is an attribute that should not be accessed from the outside.